metabrainz/messages.pot

```diff
@@ -1069,7 +1065,7 @@ msgstr ""
 msgid ""
 "Hired in the fall of 2015 as the MetaBrainz Community Manager, Freso is "
 "in charge of most of the public communication\n"
-"          channels (IRC, forums, blog, etc.) and various community "
+"          channels (Chat, forums, blog, etc.) and various community "
```

Comment:
Did you do the `messages.pot` changes by hand? I assume the `.pot` file is automatically generated by some script and we should run that instead.

Reply:
This is the only bit I do not know how to do. Happy to tackle it with guidance, otherwise I will leave it to the devs

Reply:
The info was on the readme file, but it was actually semi-incorrect and I had to play with it until I got it to work. Amended the readme in a separate commit. Also, it seems nobody had updated this since... 2017 or so? (which is why the string you were modifying here was still about Freso being in charge of communication).